preprocessors/clothes.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/clothes-detector.schema.json",
+    "type": "object",
+    "title": "Clothes Detector",
+    "description": "Preprocessor to detect the clothes worn by the individuals. The associated information can be found in preprocessor named 'ca.mcgill.a11y.image.preprocessor.clothesDetector'. ",
+    "properties":{
+        "clothes":{
+            "type":"array",
+            "description": "Contains a list of clothes worn by people",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "ID":{
+                        "description": "The ID of a person. The ID is generated by object detection preprocessor and is reused in this preprocessor",
```

Comment:
If this relies on person identification please specify this in the top-level description.